src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -223,7 +229,8 @@ public string Format()
                 { nameof(EnableQueryStringTracing), EnableQueryStringTracing },
                 { nameof(EnableDependencyTracking), EnableDependencyTracking },
                 { nameof(DependencyTrackingOptions), dependencyTrackingOptions },
-                { nameof(TokenCredentialOptions), tokenCredentialOptions }
+                { nameof(TokenCredentialOptions), tokenCredentialOptions },
+                { nameof(DiagnosticsEventListenerLogLevel), DiagnosticsEventListenerLogLevel?.ToString() }
```

Comment:
nit, end with `,` so diffs are smaller:
```suggestion
                { nameof(DiagnosticsEventListenerLogLevel), DiagnosticsEventListenerLogLevel?.ToString() },
```